### PR TITLE
tests, Makefile: fix the TSO consistency test

### PR DESCRIPTION
**Pull request by JmPotato**
### What problem does this PR solve?

Issue Number: ref #4813.

### What is changed and how does it work?

```commit-message
Fix the TSO consistency test.
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

image

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -249,6 +249,12 @@ ci-test-job: install-tools dashboard-ui
 
 TSO_INTEGRATION_TEST_PKGS := $(PD_PKG)/tests/server/tso
 
+test-tso: install-tools
```
Comment (HuSharp) [inline Makefile:252]: do we need to add in `.PHONY`?

Comment (JmPotato) [reply Makefile:252]: Nice catch. Added.

**Pull request merged**